doc/run.txt

```diff
@@ -1,13 +1,14 @@
 Synopsis
 
-    cordova-cli run [BUILD] [PROD] [TARGET] [EXP] [PLATS] [-- POPTS]
+    cordova-cli run [BUILD] [PROD] [TARGET] [EXP] [PLATS] [LIST] [-- POPTS]
```

Comment:
[LIST] should be before [EXP], and you need to define LIST below (between TARGET and EXP)